lang/luajit-openresty/Portfile

```diff
@@ -5,25 +5,22 @@ PortGroup           compiler_blacklist_versions 1.0
 PortGroup           xcode_workaround 1.0
 PortGroup           github 1.0
 
-github.setup        openresty luajit2 2.1-20210510 v
+github.setup        openresty luajit2 2.1-20220411 v
 name                luajit-openresty
-revision            1
+revision            0
+checksums           rmd160  9e77865de5b7731d4acc620695e1fb427f2d349f \
+                    sha256  d3f2c870f8f88477b01726b32accab30f6e5d57ae59c5ec87374ff73d0794316 \
+                    size    1141007
+
 categories          lang
-platforms           darwin
```

Comment:
Example Portfile on https://guide.macports.org/#development.examples still has `platforms darwin`. Why are we removing them now?

Reply:
`platforms darwin` is now the default. See https://github.com/macports/macports-base/commit/d9181903e6fda245fda3dab3bbe3dcceb2650323.